src/locales/pl.json

```diff
@@ -200,6 +200,7 @@
   },
   "next": "Dalej",
   "no-events-on-selected-day-message": "Nie ma żadnych wydarzeń w wybranym dniu",
+  "no-events-message-all": "Nie ma żadnych wydarzeń",
```

Comment:
Please sort properties alphabetically (action available in WebStorm quick action menu)